examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -265,3 +264,20 @@ void PairingCommand::OnDiscoveredDevice(const chip::Dnssd::DiscoveredNodeData &
         SetCommandExitStatus(err);
     }
 }
+
+void PairingCommand::OnCurrentFabricRemove(void * context, NodeId remoteDeviceId, CHIP_ERROR err)
+{
+    PairingCommand * command = reinterpret_cast<PairingCommand *>(context);
+    VerifyOrReturn(command != nullptr, ChipLogError(chipTool, "OnOpenCommissioningWindowCommand: context is null"));
```

Comment:
`OnOpenCommissioningWindowCommand`?  Copy/paste error?

Reply:
Oh! my mistake. I will change this log.